prinz/src/main/scala/pl/touk/nussknacker/prinz/model/Model.scala

```diff
@@ -1,16 +1,19 @@
 package pl.touk.nussknacker.prinz.model
 
-case class ModelName(name: String) {
-  override def toString: String = name
-}
+trait Model {
```

Comment:
do we have any rules when to use trait and when abstract class?

Reply:
i'm not an expert of scala but for now I think we can assume to treat traits like interfaces (no implementation) and abstract classes - something abstract with implementation based on some abstract methods that need to be implemented in subclasses. In scala it seems to bee more complicated as traits can have implementation and by them they can allow to some kind of multiple inheritance but for now I don't see any reason to use these features of traits in our model